pkg/psmdb/backup/pbm.go

```diff
@@ -445,6 +551,55 @@ func (b *pbmC) SetConfig(ctx context.Context, cfg *config.Config) error {
 	return errors.Wrap(err, "set config")
 }
 
+func (b *pbmC) ValidateBackup(ctx context.Context, cfg *config.Config, bcp *psmdbv1.PerconaServerMongoDBBackup) error {
+	if err := b.ValidateBackupInMetadata(ctx, cfg, bcp); err != nil {
+		return errors.Wrap(err, "validate backup in metadata")
+	}
+
+	if err := b.ValidateBackupInStorage(ctx, cfg, bcp); err != nil {
+		return errors.Wrap(err, "validate backup in storage")
+	}
+
+	return nil
+}
+
+func (b *pbmC) ValidateBackupInMetadata(ctx context.Context, cfg *config.Config, bcp *psmdbv1.PerconaServerMongoDBBackup) error {
```

Comment:
Here we are passing the cfg but it is not used anywhere. Most probably small mistake because the signature with `ValidateBackupInStorage` is similar. We can skip it.